### DISCUSSION TITLE
Server Pro license

### DISCUSSION THREAD

**Discussion by ronmanp** [rustdesk/rustdesk-server-pro#43] [General]
Hi, 

Could you please point me in the right direction to see the Rustdesk Server Pro license terms? We are interested in getting the business license but we want to review the terms first. 

Thank you

**Comment by Rusty-Weasel** [comment]
I like also more infos on this point, 
as i got informed that pro is only **closed-source** and not like community edition open source.

From business point of view i can understand the way with pro lic, **no problem**,  we all need to earn money,
but by this to switch from open to closed source, for customer, feels really sad, and destroy the **open source vision**.

For the moment i **stop all efforts on community project**, concerning this info,
as all suggestions/tips/etc. which i gave to your Head of Community and Sales are perhaps just flowing in a closed source product. 

Sorry but therefore i have an other understanding of "**We love Open-Source**" or further remember:
_RustDesk is a full-featured open source remote control alternative for self-hosting and security with minimal configuration._

**Therefore the people use your product,** 
- can check source code,  _<- broke in pro lic_
- can self-host,
- can check security, _<- broke in pro lic_
- can easy configure.

Are these all fixed in **Pro ?** (just cloned community server), as pro version is not available for check.
_cargo audit:_
![image](https://github.com/rustdesk/rustdesk-server-pro/assets/61587283/f31458d9-da69-4b4b-8a20-fb963a005eed)

56 possible panics - fixed in **Pro?**
![image](https://github.com/rustdesk/rustdesk-server-pro/assets/61587283/c11fbbc4-c26f-4ff3-a89e-a53d5958bce3)

As a possible customer i could expect that such upper things are fixed and prof. handled - or not ?

**Comment by Rusty-Weasel** [comment]
@rustdesk not willingness to give us a feedback ? why you move issue to discussion without comment?
